src/content/self-hosted/guides/okta/index.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Use Okta as your Authentication Provider
+description: Guide on how use Okta as the authentication provider of your Okteto Self-hosted instance
+id: identity-provider-okta
+---
+
+import Image from '@theme/Image';
+import TiersList from '@theme/TiersList';
+
+#  Use Okta as your Identity Provider<TiersList tiers="Self-Hosted" />
+
+This guide will walk you through the steps necessary to configure Okta as the authentication provider for your instace of Okteto. 
+
+Please refer to [Okta's official documentation](https://developer.okta.com/docs/concepts/oauth-openid/) on Open ID Connect for more information.
+
+## Prerequisites
+
+- An Instance of [Okteto](https://okteto.com/docs/self-hosted/).
```

Comment:
Should we call it "an instance of Okteto", or would something like "A Kubernetes cluster with Okteto installed" be better?

```suggestion
- An instance of [Okteto](https://okteto.com/docs/self-hosted/).
```